### ISSUE TITLE
Update docker to use latest python/js versions

### ISSUE THREAD

**Issue by tijptjik** [drostehk/drostehk.github.io#2]

**Comment by tijptjik** [comment]
```
FROM continuumio/anaconda3:4.1.1
MAINTAINER Bill McCord <bill@droste.hk>

RUN apt-get update && apt-get install -y curl make

RUN curl -sL https://deb.nodesource.com/setup_4.x | /bin/bash -

RUN apt-get install -y nodejs

RUN npm install --global gulp-cli

COPY ./ /var/www/drostehk

WORKDIR /var/www/drostehk

RUN conda env create -n drostehk -f environment.yml

EXPOSE 8000

ENTRYPOINT ["/bin/bash", "-c", "set -x && ./start_dev.sh"]
```

Probably needs to be 

```
FROM continuumio/anaconda3:4.1.1
MAINTAINER Bill McCord <bill@droste.hk>

RUN apt-get update && apt-get install -y curl make

RUN curl -sL https://deb.nodesource.com/setup_4.x | /bin/bash -

RUN apt-get install -y nodejs

COPY ./ /var/www/drostehk

WORKDIR /var/www/drostehk

RUN npm install

RUN python3 -m venv drostehk-env && source drostehk-env/bin/activate

RUN pip install -U pip

RUN pip install -r requirements.txt

EXPOSE 8000

ENTRYPOINT ["/bin/bash", "-c", "set -x && ./start_dev.sh"]
```

**Comment by tijptjik** [comment]
Fixed by 71c95e845e5fc46b1e610b49f562034c6c29edd7
